### DISCUSSION TITLE
New document field columns responsive

### DISCUSSION THREAD

**Discussion by Temkit** [keystonejs/keystone#5496] [Feature requests]
The new document field is definitely something. I started using it on anew project, but I see columns are not responsive !! Is there a way ?
![Screenshot_2021-04-20-08-03-17-02](https://user-images.githubusercontent.com/13275723/115352137-e9926900-a1ae-11eb-83ee-a8aaa6d2892e.jpg)

**Comment by gautamsi** [comment]
if you are rendering it in some page, you can have your custom renderer with responsive. Ideally this should be responsive by default.

**Reply by bladey** [reply]
Hey @Temkit, @gautamsi is right here, it is up to the developer on how this is rendered in your own front-end app. Let me know if you have any questions!
